src/services/permissionsHelper/hooks/baseServicesAccess.js

```diff
@@ -1,18 +1,34 @@
 const { Forbidden } = require('@feathersjs/errors');
+const { copyParams } = require('../../../global/utils');
 
 const baseServicesAccess = path => async (context) => {
-	const { params, app } = context;
+	const { params, app, method } = context;
 	if (!(params.provider === 'rest')) {
 		return context;
 	}
 
-	const copyParams = Object.assign({}, params);
-	copyParams.route.ressourceId = context.id;
-	const { access } = await app.service(path).get(params.user.id, copyParams);
-	if (!access) {
-		throw new Forbidden('You have no access.');
+	const copyParamsWithQuerys = Object.assign({}, params);
+
+	if (method === 'find') {
```

Comment:
ist das nicht schon ein seperater hook? wird bei find auggrufen und gibt einen wert zurück, so das der code danach in dem fall nicht ausgeführt wird und der codeteil im if wird nur bei find ausgeführt?

Reply:
Ja stimmt, sollte in 2 hooks aufgesplittet werden.

Reply:
todo